kolena/dataset/dataset.py

```diff
@@ -99,7 +99,11 @@ def _normalize_url(x: str) -> str:
 
 
 def _infer_datatype_value(x: str) -> str:
```

Comment:
i think we need to update the type hint to reflect the change

Reply:
With the "implied" permissiveness of DataFrame input could be heterogeneous, in reality we're handling `Any`.